core/src/test/scala/scala/pickling/issues/README.md

```diff
@@ -0,0 +1,4 @@
+# Which tests should I write here?
+
+These tests represent bugs that have been filed in the issue
```

Comment:
I like these distinctions, with the caveat that perhaps these should be called 'regression tests' to prevent bugs from reopening.